infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -68,6 +68,9 @@
       "plugins": {{ to_plugin_config.keys() | list | to_json }},
       "plugin_config" : {{ to_plugin_config | to_nice_json(indent=2) }},
       "traffic_vault_backend": "{{ to_traffic_vault_backend }}",
+      "tls_config": {
+         "ClientAuth": "{{ to_go_client_auth_enabled }}"
+      },
```

Comment:
Also needs

```json
        "MinVersion": 769,
```

Reply:
Is there no default in TO that I am leaving this up to? 

What does the tls_config get set to without this block as it was before?

Reply:
> Is there no default in TO that I am leaving this up to?

Not currently, no

> What does the tls_config get set to without this block as it was before?

Whatever the TLS config is, empty or no, is used as the server's TLS config. The defaults are internal Go HTTP server logic

https://github.com/apache/trafficcontrol/blob/708a95cd87334ca6da6dc75c3849e00b995be81a/traffic_ops/traffic_ops_golang/traffic_ops_golang.go#L191-L193